.releaserc

```diff
@@ -1,8 +1,21 @@
 {
   "tagFormat": "${version}",
+  "branches": [
+    { "name": "master", "prerelease": false },
+    { "name": "development", "prerelease": "staging" }
```

Comment:
"staging" as a term to describe infrastructure. Because this is a library it is not directly part of any infrastructure. Better to call this "development"